graphql_test.go

```diff
@@ -3018,23 +3100,65 @@ func TestErrorPropagation(t *testing.T) {
 	})
 }
 
+type ambiguousResolver struct {
+	Name string // ambiguous
+	University
+}
+
+type University struct {
+	Name string // ambiguous
+}
+
+func TestPanicAmbiguity(t *testing.T) {
```

Comment:
Can you, please, move `ambiguousResolver` and `University` inside of the test (e.i. `TestPanicAmbiguity`)?